src/lib/Components/Bidding/Components/Title.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { TextProperties } from "react-native"
+import styled from "styled-components/native"
+
+import { Fonts } from "../../../data/fonts"
+
+export class Title extends React.Component<TextProperties> {
+  render() {
+    return <StyledText {...this.props}>{this.props.children}</StyledText>
+  }
+}
+
+const StyledText = styled.Text`
+  font-family: "${Fonts.GaramondRegular}";
```

Comment:
Realized the font in designs is `GaramondSemibold`. I found the font files in the assets directory and adding it like this into `fonts.ts` worked:
```
export enum Fonts {
  ...
  GaramondSemibold = "AGaramondPro-Semibold",
  ...
}

export default {
  ...
  "garamond-semibold": "AGaramondPro-Semibold",
  ...
}
``` 
Not sure if this is all that is needed to use the font. @orta?

Reply:
@sepans Let's pair on this as a separate PR since this is going to be a new public API.

Reply:
We will send a separate PR for the fonts and stuff we worked on, but now that https://github.com/artsy/emission/pull/1006 is in I'll have to rebase and update this PR slightly.

Reply:
:+1: